builder_data.yaml

```diff
@@ -24,8 +24,8 @@ metrics_url: http://metrics.delivery.puppetlabs.net/overview/metrics
 benchmark: false
 staging_server: weth.delivery.puppetlabs.net
 signing_server: mozart.delivery.puppetlabs.net
-yum_host: 'yum.puppetlabs.com'
-apt_host: 'apt.puppetlabs.com'
+yum_host: 'saturn.delivery.puppetlabs.net'
+apt_host: 'saturn.delivery.puppetlabs.net'
```

Comment:
nit: should we use enterprise.delivery (the service name) instead of the hostname?

Reply:
ahh yeah, that's a good idea, will update